docs/software/commands.md

```diff
@@ -638,3 +638,58 @@ enabled by specifying a port via the HTTP_QUERY_PORT config setting.
   value of the `TTL` entry to the current ledger sequence number.
 
   `ledgerSeq` gives the ledger number on which the query was performed.
+
+* **`getledgerentry`**<br>
+  A POST request with the following body:<br>
+
+  ```
+  ledger=NUM&k=Base64&k=Base64...
+  ```
+- `ledger`: An optional parameter, specifying the ledger snapshot to base the query on.
```

Comment:
In https://github.com/stellar/stellar-core/pull/4623#issuecomment-2689743271 provided some feedback about the naming of `ledger` and `ledgerSeq` inconsistency in the req and resp of `getledgerentry`. Looks like that was addressed and `ledger` is now used consistently in the request and the response, but that makes the endpoint inconsistent with `getledgerentryraw` that uses `legderSeq` consistently.

Can we use `ledgerSeq` consistently across all the query endpoints, or is the other endpoint deprecated now that this one is out and therefore doesn't matter?